nucypher/blockchain/eth/token.py

```diff
@@ -48,21 +48,24 @@ class NU:
     An amount of NuCypher tokens that doesn't hurt your eyes.
     Wraps the eth_utils currency conversion methods.
 
-    The easiest way to use NU, is to pass an int, float, or str, and denomination string:
+    The easiest way to use NU, is to pass an int, Decimal, or str, and denomination string:
 
     Int:    nu = NU(100, 'NU')
     Int:    nu_wei = NU(15000000000000000000000, 'NuNit')
```

Comment:
Remove reference to nu_wei; use nunits instead

```suggestion
    Int:    nunits = NU(15000000000000000000000, 'NuNit')
```